skbio/io/_exception.py

```diff
@@ -130,3 +130,9 @@ class BIOMFormatError(Exception):
     """Raised when a ``BIOM`` formatted file cannot be parsed."""
 
     pass
+
+
+class EmbeddingFormatError(Exception):
+    """Raised when an ``Embedding`` formatted file cannot be parsed."""
```

Comment:
I'm following the formatting of the other errors -- if this will be changed, we will need to change the rest of the error docs (which I think we should defer to another PR).